webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -41,4 +45,21 @@ export const updateContentView = (cvId, params) => async dispatch => dispatch(pu
   },
 }));
 
+export const getContentViewRepositories = (cvId, params, status) => {
+  const apiParams = { ...params };
+  let apiUrl = `/content_views/${cvId}/repositories`;
+  if (status[ADDED] && status[NOT_ADDED]) {
+    apiUrl += '/show_all';
+  } else if (status[NOT_ADDED]) {
+    apiParams.available_for = 'content_view';
+  }
+
+  return get({
+    type: API_OPERATIONS.GET,
+    key: cvDetailsRepoKey(cvId),
+    url: api.getApiUrl(apiUrl),
+    params: apiParams,
```

Comment:
@jeremylenz I updated this from `params,` to `apiParams,` but forgot the key is still named `params` :man_facepalming:

This fixes most of search and dropdown combination issues, but the search is still not working when the dropdowns are changed after an intial search, I'm looking more into that.